dialog/en-us/connection.error.dialog

```diff
@@ -0,0 +1 @@
+Connection to mycroft.ai failed, pairing code could not be fetched. Check your internet connection and try again.
```

Comment:
Where does this message get spoken?  I don't see any reference to connection.error in the code.

Reply:
Good thing you noticed this. Have inserted the speak_dialog call for it.